protected/humhub/modules/space/resources/js/humhub.space.js

```diff
@@ -70,6 +71,13 @@ humhub.module('space', function (module, require, $) {
         }
     };
 
+    var requestMembershipSend = function(event) {
```

Comment:
See `admin.setAsDefault` comment.

Reply:
same here, we need post request.  With naming i think it's ok in this case, it's space.js